lavamoat/browserify/beta/policy.json

```diff
@@ -1807,8 +1799,8 @@
     },
     "@metamask/remote-feature-flag-controller": {
       "packages": {
-        "@metamask/remote-feature-flag-controller>@metamask/base-controller": true,
```

Comment:
@MetaMask/policy-reviewers the changes comes from controller update, 
- use new `@metamask/controller-utils` within controller to access packages
- update distribution type.

Thanks for reviewing ❤️